site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "High Availability Agones"
+date: 2023-02-08T15:02:23Z
+weight: 20
+description: >
+  Allows controllers to be strictly for control loops; feature separates extensions and webhooks out of your controllers
+---
+
+## High Availability for Agones Controller
+
+The `agones-controller` pod is currently not only used for control loops of game server related resources, but also include other services such as webhooks and allocation extension srvices. High availability agones mean as opposed to having controller host everything mentioned, we will add additional `extensions` pods that exposes the webhook and allocations service. Thus, making `agones-controller` pod strictly for control loops. Read more about pod disruption [here](https://agones.dev/site/docs/advanced/controlling-disruption/)
+
+{{< alpha title="`split controller and extensions` API" gate="SplitControllerAndExtensions" >}}
+
+{{< alert title="GKE Autopilot Clusters" color="warning" >}}
+In order to use GKE Autopilot clusters with Agones, this feature flag `SplitControllerAndExtensions` must be enabled along with `SafeToEvict` (should already be enabled). 
+
+More info on `SafeToEvict` [here](https://agones.dev/site/docs/advanced/controlling-disruption/#disruption-in-kubernetes)
+{{< /alert >}}
+
+## Extension Pod Configrations 
+
+Current setup has `extensions` pods inherit values through the `controller` values within the `values.yaml` through **yaml inheritance**. This means that changes made to the `controller` configurations within the file will have a direct effect on the `extensions` values. Essentially acts as a copy & paste. To override the configurations within `values.yaml`, declare and initialize the desired key and value within the `extensions` section within `values.yaml`. For example, lets change the `apiServerQPS` field value inhertied for `extensions` to 600 instead of the original 400:  
```

Comment:
This part is confusing as most end-users won't be touching `values.yaml` and will instead be using an override yaml or `--set`. I would rework this section to focus on the end-user perceived behavior as if they weren't modifying `values.yaml` directly. You can add a comment to `values.yaml` if you feel passionately about documenting it.